skillmap/src/lib/core.ts

```diff
@@ -0,0 +1,25 @@
+import { pushNotificationMessage } from "react-common/Notification"
+
+///////////////////////////////////////////////////////////
```

Comment:
The name of this file could be more descriptive. Maybe `notificationService.ts`? or `notifications.ts`? `toast.ts`?

Reply:
oh yeah I was just copying the structure over in the webapp in case we....ever wanted to match more things between the two? But I'm also happy  to make this a notifications only file

Reply:
yeah let's do notifications only. we've mostly been doing 1 component per file in skillmap

Reply:
👍